Library/RemoteConfig/RemoteConfigFeature+Helpers.swift

```diff
@@ -1,10 +1,12 @@
 /// Return remote config values either a value from the cloud, if it found one, or a default value based on the provided key
+private func featureEnabled(feature: RemoteConfigFeature, defaultValue: Bool = false) -> Bool {
+  let valueFromDefaults = AppEnvironment.current.userDefaults
+    .remoteConfigFeatureFlags[feature.rawValue]
 
-private func featureEnabled(feature: RemoteConfigFeature) -> Bool {
-  return AppEnvironment.current.userDefaults
-    .remoteConfigFeatureFlags[feature.rawValue] ??
-    (AppEnvironment.current.remoteConfigClient?
-      .isFeatureEnabled(featureKey: feature) ?? false)
+  let valueFromRemoteConfig = AppEnvironment.current.remoteConfigClient?
+    .isFeatureEnabled(featureKey: feature)
+
+  return valueFromDefaults ?? valueFromRemoteConfig ?? defaultValue
```

Comment:
Slight refactoring to support new default values, and to make this easier to read and debug.